src/components/TabbedContent/TabbedContent.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'react-content-loader';
+import TabBar from '../TabBar';
+
+class RenderContent extends React.Component {
+  static propTypes = {
+    skeleton: PropTypes.bool,
+    content: PropTypes.shape({}),
+  };
+
+  state = {
+    render: false,
+  };
+
+  componentDidMount() {
+    requestAnimationFrame(() => {
```

Comment:
I'm not entirely sure why, but requestAnimationFrame has to be called twice for this to work.

Reply:
weird, but if it works :)

Curious to know what's going on though

Reply:
wonder if just deferring it with a setTimeout would work as well

Reply:
yeah it would, but I think using requestAnimationFrame twice has the least amount of delay.
If I figure out how exactly this works, I'll let you know.